lib/src/device.dart

```diff
@@ -1,20 +1,17 @@
 import 'dart:async';
-import 'dart:ui';
-
 import 'package:flutter/foundation.dart';
 import 'package:flutter/services.dart';
 
-import 'extensions/vector2.dart';
-
-/// Some utilities that did not fit anywhere else.
+/// Provides methods for controlling the device (e.g. setting the screen to fullscreen).
 ///
-/// To use this class, access it via [Flame.util].
-class Util {
+/// To use, access it via [Flame.device]
+class Device {
   /// Sets the app to be fullscreen (no buttons bar os notifications on top).
   ///
   /// Most games should probably be this way.
```

Comment:
I know this comment was already existing. But I think we can just remove this. It is up to the developer to decide that. This feels a bit judging.. "If you dont do this, you are doing it wrong". And it does not really provide any useful information about the method itself.

Reply:
Agree! Removed!